### DISCUSSION TITLE
How did my GitHub Student Developer Pack request get approved after five rejections?

### DISCUSSION THREAD

**Discussion by mirzaabdullaht** [community/community#143960] [GitHub Education]
### Hi everyone,

<img width="215" alt="image" src="https://github.com/user-attachments/assets/5b20b366-f22a-4b08-86b9-40db17977a43">
<img width="260" alt="image" src="https://github.com/user-attachments/assets/744546c2-0cd3-47eb-b4a6-d5700354a266">

I wanted to share my experience with getting the GitHub Student Developer Pack. I ran into the same problem many of you are facing - there was no option to upload documents, and when I tried using the live camera, my application kept getting rejected.

**Here's what I did:**

- I tried applying from different devices
- I emailed GitHub's customer support
- I even used a MacBook because its camera is way better than other laptops.
- I also followed the rejection reason and applied to my account e.g i editted my name same as same as was mentioned on my student ID card.

My student ID was a bit tricky because it had two pages - one with my name and the other with the expiry date. After a bunch of rejections and following their suggestions, I gave it another shot with the MacBook. It took 5 rejections over a few days, but on my 6th try, it finally worked!

Now I'm just waiting for the benefits to kick in, which should take about 3 days.

**Some tips from my experience:**

- Don't give up. Keep trying until you get approved.
- Make sure to apply before you graduate. I only had 2 months left on my student ID validity.
- Try to get the clearest picture possible of your ID and with your live face.

I know how frustrating it is when you're trying to upload a clear document but can't, and you keep getting rejected. But hang in there!

My GitHub education request finally got approved.

Good luck to all of you trying to get your GitHub Education Pack :)

**Comment by huddmax** [comment]
I also tried several times until I finally got approved! But now I've been approved, and today I received my GitHub Foundations certificate.
